Python in mantid (#28806)
mantidproject/mantid: docs/source/tutorials/python_in_mantid/script_generation/04_pim_ex_2.rst

.. _04_pim_ex_2:

============================
Python in Mantid: Exercise 2
============================

The aim of this exercise is to create a general script that can accept user interaction to alter the parameters for future uses

Perform a set of algorithms by pointing and clicking within Mantid
Use the history window to generate a script based upon the executed algorithms
Use the dialog functions to make the script more general for future runs.

.. contents:: Table of contents
    :local:

A - Processing ISIS Data
========================

Running an analysis Manually
----------------------------

> Here you are going to perform a number of steps in the Mantid user interface to correct some LOQ data by the incident beam monitor. For more information in a specific algorithm, click on its name.

Reviewer: ```suggestion
Here you are going to perform a number of steps in the Mantid user interface to correct a LOQ dataset using the incident beam monitor. For more information in a specific algorithm, click on its name.
```
